palindrome of intiger

```diff
@@ -0,0 +1,29 @@
+#include <bits/stdc++.h>
+using namespace std;
+int palindrome(int n,int b){
+    int s=0;
```

Comment:
Why Palindrome function type is int. Also why does it take 2 parameters n,b. Function should return true/false based on whether it is palindrome or not